bot/exts/utilities/githubinfo.py

```diff
@@ -66,7 +257,7 @@ async def github_user_info(self, ctx: commands.Context, username: str) -> None:
 
             embed = discord.Embed(
                 title=f"`{user_data['login']}`'s GitHub profile info",
-                description=f"```\n{user_data['bio']}\n```\n" if user_data["bio"] else "",
+                description=f"```{user_data['bio']}```\n" if user_data["bio"] else "",
```

Comment:
Hmm, do we not need to keep the initial and last newline? I recall that some codeblocks get weird formatting if you drop it. Could be wrong.

Reply:
That's correct, we should have them here.

Reply:
Right, changed in https://github.com/python-discord/sir-lancebot/pull/778/commits/c815a19612be9e0a28403786696964e14420204f.